editor/plugins/visual_shader_editor_plugin.cpp

```diff
@@ -186,20 +186,20 @@ bool VisualShaderEditor::_is_available(int p_mode) {
 
 		int temp_mode = 0;
 
-		if (p_mode & VisualShader::TYPE_FRAGMENT) {
+		if (p_mode & TYPE_FLAGS_VERTEX) {
+			temp_mode |= 1;
```

Comment:
I'm not familiar with the big picture here, but doesn't all this code just end up copying `p_mode` in `temp_mode`?

Reply:
Ah, really, Sorry for the sooner merge, I will remove that in the later commits.

Reply:
I didn't check if that's true though, it's just an intuition from reading the diff.

Reply:
It's true - I've checked :)